### PR TITLE
Created user-manual.md

### PR DESCRIPTION
**Pull request by thaowii**

### CODE REVIEW

File: user-manual.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# What is RateMyCSE?
+RateMyCSE is a course reviewing website specifically for the students of the University of Washington studying computer science. It is a place where you can find everything you need on a CSE class in order to build a balanced schedule. You can read written reviews from students who have taken the class, the average difficulty, workload, and practicality ratings of each class, see tags for classes that have coding, exams, or group projects (work in progress). You can also provide insight for other students by leaving your own reviews and ratings.
+
+# How to install and run RateMyCSE
+RateMyCSE is a free, public website. You can access it by visiting rate-my-cse.vercel.app
```
Comment (celestinebuendia) [inline user-manual.md:5]: Nitpick: can you turn the link into a clickable link using the '[rate-my-cse.vercel.app](rate-my-cse.vercel.app)' link format

**Review by celestinebuendia — CHANGES_REQUESTED** [review]
Looks great! Thanks for doing all this. Small nitpick below though.

File: user-manual.md (hunk 1 above)
Comment (thaowii) [reply user-manual.md]: Oop woops I thought it was ! Fixed.

**Review by celestinebuendia — APPROVED** [review]
Thanks for the change! Sorry for the delay!

**Pull request merged**